### PR TITLE
[AIDAPP-211]: Refactor the primary url of the application portal

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-211

### Technical Description

Refactor the primary url of the application portal

### Any deployment steps required?

yes, run below commands,
npm run build
if any error will come because of tip top editor then first run composer ci  and then npm run build again.

### Are any Feature Flags Added?

No
_______________________________________________

### CODE REVIEW

**Review by joelicatajr — CHANGES_REQUESTED** [review]
@ketan-canyon @Orrison AIDAPP-211 calls for refactoring the baseUrl to `/portal` rather than `/portals`.

**Comment by ketan-canyon** [comment]
> @ketan-canyon @Orrison AIDAPP-211 calls for refactoring the baseUrl to `/portal` rather than `/portals`.

Done.. @joelicatajr

File: app-modules/portal/routes/web.php [hunk 1]
```diff
@@ -41,30 +41,35 @@
...
     ->middleware([
         'web',
         EnsureFrontendRequestsAreStateful::class,
     ])
     ->group(function () {
+        Route::get('/knowledge-management/{any?}', function ($any = null) {
+            $url = $any ? '/portal/' . $any : '/portal';
+
+            return redirect($url, 301);
+        })->where('any', '.*');
```
Comment (Orrison) [inline app-modules/portal/routes/web.php:55]: I believe this is going to need to be moved out of this prefix group and target `/portals/knowledge-management/{any?}` now that we have changed the prefix above.

Without doing so, redirects from the old URL structure do not work.

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/portal/routes/web.php (hunk 1 above)
Comment (Orrison) [reply app-modules/portal/routes/web.php]: Took care of this in [bf6a574](https://github.com/canyongbs/aidingapp/pull/204/commits/bf6a574a7a04a891890c3587ed8c2705c0ef9316)

**Review by Orrison — APPROVED** [review]

**Pull request merged**